### PR TITLE
Taco List View

### PR DESCRIPTION
**Pull request by tolkiana**
Creates Taco list with hardcoded Data
screen shot 2017-08-23 at 12 04 45 am

### CODE REVIEW

**Comment by macklinu** [comment]
Awesome! I'll give this a review later today. 😄

File: app/src/main/java/com/tolkiana/tacoapp/TacoListActivity.kt [hunk 1]
```diff
@@ -2,11 +2,22 @@ package com.tolkiana.tacoapp
 
 import android.support.v7.app.AppCompatActivity
 import android.os.Bundle
+import android.support.v7.widget.LinearLayoutManager
+import kotlinx.android.synthetic.main.activity_taco_list.*
+import java.sql.Array
```
Comment (macklinu) [inline app/src/main/java/com/tolkiana/tacoapp/TacoListActivity.kt]: This looks to be unused.

File: app/src/main/java/com/tolkiana/tacoapp/TacoListActivity.kt [hunk 2]
```diff
@@ -2,11 +2,22 @@ package com.tolkiana.tacoapp
...
+import java.util.*
 
 class TacoListActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_taco_list)
+
+        val tacoList = Arrays.asList(Product("Pastor"), Product("Canasta"), Product("Asada"))
```
Comment (macklinu) [inline app/src/main/java/com/tolkiana/tacoapp/TacoListActivity.kt]: Cool! This is a simple way to get some data into the adapter and work on the list view. 👍

File: app/src/main/java/com/tolkiana/tacoapp/ProductAdapter.kt [hunk 3]
```diff
@@ -16,7 +16,7 @@ class ProductAdapter(private val products: List<Product>): Adapter<ProductAdapte
 
     override fun onBindViewHolder(productHolder: ProductHolder?, position: Int) {
         val product = products[position]
-        productHolder?.productNameTextView?.text = product.getName()
+        productHolder?.productNameTextView?.text = product.name
```
Comment (macklinu) [inline app/src/main/java/com/tolkiana/tacoapp/ProductAdapter.kt:19]: I like that getters in Kotlin are more like properties (`getName()` becomes `name`). 🙌

**Review by macklinu — COMMENTED** [review]
This is looking like a solid start. 👍 Loading JSON data seems very close!

I'm having trouble getting my emulator working, so I'll see if I can get an Android device charged and use that for our mentorship meetings so that I can build this to a device.

File: app/src/main/java/com/tolkiana/tacoapp/ProductAdapter.kt (hunk 3 above)
Comment (tolkiana) [reply app/src/main/java/com/tolkiana/tacoapp/ProductAdapter.kt:19]: :)

**Pull request merged**